### ISSUE TITLE
perturbations.make_perturbed_fun has incorrectly scaled gradients

### ISSUE THREAD

**Issue by adconner** [google-deepmind/optax#1308]
My first experiments with make_perturbed_fun reveal that the gradient calculation is incorrect. With the identity function and normal noise, the perturbed function is mathematically the identity function (so we get an unbiased estimator). Its Jacobian then, should be an unbiased estimator of the identity matrix. What we get seems to be off by a factor of sigma.

```
In[73]: f=optax.perturbations.make_perturbed_fun(lambda x: x, sigma=0.1, noise=optax.perturbations.Normal())

In [75]: f(jnp.array(1.0), jax.random.PRNGKey(1))
Out[75]: Array(0.998676, dtype=float32)

In [76]: jax.grad(f)(jnp.array(1.0), jax.random.PRNGKey(1))
Out[76]: Array(0.08708793, dtype=float32)

In [77]: jax.grad(f)(jnp.array(0.0), jax.random.PRNGKey(1))
Out[77]: Array(0.10032693, dtype=float32)

In [78]: f2=optax.perturbations.make_perturbed_fun(lambda x: x, sigma=0.5, noise=optax.perturbations.Normal())

In [79]: f2(jnp.array(1.0), jax.random.PRNGKey(1))
Out[79]: Array(0.9933809, dtype=float32)

In [80]: jax.grad(f2)(jnp.array(1.0), jax.random.PRNGKey(1))
Out[80]: Array(0.48839524, dtype=float32)

In [81]: jax.grad(f2)(jnp.array(0.0), jax.random.PRNGKey(1))
Out[81]: Array(0.50163394, dtype=float32)

In [83]: f2(jnp.ones(4), jax.random.PRNGKey(1))
Out[83]: Array([0.9933809, 1.0099303, 1.0249068, 1.0073029], dtype=float32)

In [84]: jax.jacobian(f2)(jnp.ones(4), jax.random.PRNGKey(1))
Out[84]:
Array([[ 4.8839524e-01,  1.3756592e-02,  3.2798059e-02,  2.7877303e-02],
       [-1.9342428e-02,  5.2607626e-01,  5.0115172e-02,  2.5842471e-02],
       [-3.0254519e-02,  2.0161595e-02,  5.3830910e-01,  2.4624784e-02],
       [ 3.2641597e-05,  3.1096857e-02,  5.9832685e-02,  5.2487993e-01]],      dtype=float32)

```

**Comment by carlosgmartin** [comment]
Since https://github.com/google-deepmind/optax/pull/1310 has been merged, I think this issue can be closed.

**Comment by rdyro** [comment]
Fixed by [https://github.com/google-deepmind/optax/pull/1310](https://github.com/google-deepmind/optax/pull/1310)
